src/rest_api.py

```diff
@@ -1,11 +1,26 @@
+import os
 import flask
+import logging
 from flask import Flask, request
 from flask_cors import CORS
 from recommender import RecommendationTask
 from stack_aggregator import StackAggregator
 from stack_recommender import StackRecommender
 
+
+def setup_logging(flask_app):
+    """Perform the setup of logging (file, log level) for this application."""
+    if not flask_app.debug:
```

Comment:
Just wondering, what `if flask_app.debug` ? The logging is not set up in that case ? Or how does it work ?

Reply:
Flask should set logging level to DEBUG when running in debug mode. We don't want to override that.